arbitrum-docs/stylus/how-tos/adding-support-for-new-languages.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: 'Adding Stylus support for new programming languages'
+sidebar_label: 'Adding support for new programming languages'
+description: 'A guide on adding support for more programming languages to Stylus'
```

Comment:
```suggestion
description: 'Stylus shipped with native support for Rust and C, but you can add support for any programming language that compiles to WebAssembly. This how-to walks you through the process of extending Stylus with Zig support, allowing you to write EVM-compatible Zig smart contracts.'
```

Reply:
meta -> `description` is what appears in search & impacts SEO, so we can have some fun targeting specific keywords and phrases, making it inviting, etc. Not sure if we're comfortable saying "extending", nits welcome :)